src/androidTest/java/com/github/aloomaio/androidsdk/aloomametrics/DecideFunctionalTest.java

```diff
@@ -55,7 +56,7 @@ public SharedPreferences get(final long timeout, final TimeUnit unit) throws Int
         };
 
         mExpectations = new Expectations();
-        mMockPoster = new ServerMessage() {
+        mMockPoster = new HttpService() {
             @Override
```

Comment:
There is a weird error reported in this line:

"Method does not override or implement method form the super class"
`@Override
            public byte[] performRequest(`